hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1384,7 +1386,15 @@ RegionSpaceUse convertRegionSize(RegionInfo regionInfo, Long sizeInBytes) {
     } else {
       serverLoad.setInfoServerPort(-1);
     }
-
+    MetricsUserAggregateSource userSource =
+        metricsRegionServer.getMetricsUserAggregate().getSource();
+    if (userSource != null) {
+      UserLoad.Builder userLoadBldr = UserLoad.newBuilder();
```

Comment:
This should be in createUserLoad(). It looks like it's causing wrong metric values.